st_copy_to_clipboard/__init__.py

```diff
@@ -32,6 +50,10 @@ def main():
     st.write("## Example")
     text = st.text_input("Enter text to copy to clipboard", value="Hello World")
     st_copy_to_clipboard(text)
+    st_copy_to_clipboard(text, before_copy_label='📋Push to copy', after_copy_label='✅Text copied!')
+    st_copy_to_clipboard(text, before_copy_label='Push to copy', after_copy_label='Text copied!', show_text=True)
```

Comment:
Hi @mmz-001 !
Thank you for the awesome component.
I make two modifications that might be useful for others. It just works.
Would be great to push it to main and to make it available it pip package